images/primary_node/repo_url_insert_script.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+# -*- coding: utf-8 -*-
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script helps to insert correct repo URLs for the MapR.
+
+MAPR_VERSION="$1"
+MEP_VERSION="$2"
+
+if [ "$MAPR_VERSION" = "" ]; then
```

Comment:
I don't think we need this kind of command validation. I'd remove it to avoid the need for upkeep.